agent/php_agent.h

```diff
@@ -902,6 +902,22 @@ extern const char* nr_php_zend_execute_data_scope_name(
  */
 extern uint32_t nr_php_zend_execute_data_lineno(
     const zend_execute_data* execute_data);
+
+/*
+ * Purpose : Return a uint32_t (zend_uint) line number value of zend_function.
+ *
+ * Params  : 1. zend_function.
+ *
+ * Returns : uint32_t lineno value
+ *
+ */
+static inline uint32_t nr_php_zend_function_lineno(const zend_function* func) {
+  if (NULL != func) {
+    return func->op_array.line_start;
```

Comment:
Could `op_array` be NULL?

Reply:
It's not a pointer.
https://github.com/infusion/PHP/blob/master/Zend/zend_compile.h#L282